<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD:hiwonder_qt/mainwindow.ui
    <width>800</width>
    <height>591</height>
=======
    <width>950</width>
    <height>765</height>
>>>>>>> 38cefc28371631f4e72e6df4dc3f8da3f848f4e9:mainwindow.ui
   </rect>
  </property>
  <property name="windowTitle">
   <string>main</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>278</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comL"/>
    </item>
    <item>
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>OPEN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>CLOSE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="clampButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CLAMP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_1_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_2_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>690</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_3_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>690</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_4_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>690</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_5_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>690</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_6_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>690</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_posButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SET POSITION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stand_upButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stand UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="S1_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="S2_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="S3_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="S4_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="S5_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="S6_label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sitButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getBoxButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Box</string>
   </property>
  </widget>
  <widget class="QSlider" name="S6_verSlider">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>400</y>
     <width>21</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSlider" name="S5_verSlider">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>400</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S4_verSlider">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S3_verSlider">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S2_verSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S1_verSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="socketButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Detetor</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="servo_1_spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::down-button{
    width: 50
}
QSpinBox::up-button{
    width: 50
}
</string>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="servo_2_spinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::down-button{
    width: 50
}
QSpinBox::up-button{
    width: 50
}
</string>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="servo_3_spinBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::down-button{
    width: 50
}
QSpinBox::up-button{
    width: 50
}
</string>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="servo_4_spinBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::down-button{
    width: 50
}
QSpinBox::up-button{
    width: 50
}
</string>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="servo_5_spinBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::down-button{
    width: 50
}
QSpinBox::up-button{
    width: 50
}
</string>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="servo_6_spinBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::down-button{
    width: 50
}
QSpinBox::up-button{
    width: 50
}
</string>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="All_Servos_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>28</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
